Change ansible-lint version since version 6.0.0 and greater doesn't work

opeco17 opened the pull request:
Closes #3498

molecule with version 3.4 depends on `prerun` module in `ansible-lint` but `ansible-lint` with version 6.0.0 and greater doesn't have `prerun` module.
We should restrict the version of `ansible-lint`.

ssbarnea commented:
To make this work you need to also add a ceiling on pre-commit config files that are now failing,... due to the same reason.

opeco17 commented:
Hi, @ssbarnea 
I've modified the version of ansible-lint in pre-commit hook

Since there is a dependency issue in black in pre-commit hook and it doesn't work correctly, I've fixed it in the new PR (https://github.com/ansible-community/molecule/pull/3508)
